hooks/tk-houdini_actions.py

```diff
@@ -116,4 +137,46 @@ def _merge(self, path, sg_publish_data):
         hou.hipFile.merge(path,
                           node_pattern="*",
                           overwrite_on_conflict=False,
-                          ignore_load_warnings=False)       
+                          ignore_load_warnings=False) 
+
+
+    ##############################################################################################################
+    def _import(self, path, sg_publish_data):
+
+        import hou
+        app = self.parent
+
+        name = sg_publish_data.get("name", "alembic")
+        path = self.get_publish_path(sg_publish_data)
+
+        obj_context = hou.node("/obj")
```

Comment:
I modified the code to first attempt to retrieve the current obj context and create the node there. If it can't create it there it falls back to top-level "/obj". Will resubmit for CR so you can see if what I did makes sense.